app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">
```

Comment:
ok, Do the other items required too? I think you can delete them. Can yo test it?